File: pyg_cart_web/src/main/webapp/js/service/userService.js
app.service("userService",function ($http) {
    this.sendCode=function (phone) {
      return  $http.get("./user/sendCode/"+phone)
    };
 this.register=function (user,code) {
      return  $http.post("./user/register/"+code,user)
    }

    this.showName=function () {
     return $http.get("./user/showName")
    }

});

File: pyg_user_interface/src/main/java/com/pyg/user/service/UserService.java
package com.pyg.user.service;

import com.pyg.pojo.TbUser;

/**
 * @author dibulidubulijiojio
 * @version v1.0
 * @date 2019/3/28  9:40
 * @description TODO
 **/
public interface UserService {

    void sendCode(String phone);

    void register(TbUser user, String code);
}
